src/community/ogcapi/ogcapi-core/src/main/java/org/geoserver/ogcapi/FreemarkerTemplateSupport.java

```diff
@@ -45,27 +52,37 @@ public FreemarkerTemplateSupport(GeoServerResourceLoader loader) {
 
     /**
      * Returns the template for the specified feature type. Looking up templates is pretty
-     * expensive, so we cache templates by feture type and template.
+     * expensive, so we cache templates by feature type and template.
      */
     public Template getTemplate(ResourceInfo resource, String templateName, Class<?> clazz)
             throws IOException {
-        GeoServerTemplateLoader templateLoader =
-                new GeoServerTemplateLoader(clazz, resoureLoader) {
-                    @Override
-                    public Object findTemplateSource(String path) throws IOException {
-                        Object source = super.findTemplateSource(path);
-                        if (source == null) {
-                            source = rootLoader.findTemplateSource(path);
-
-                            // wrap the source in a source that maintains the original path
-                            if (source != null) {
-                                return new ClassTemplateSource(path, source);
-                            }
-                        }
-
-                        return source;
-                    }
-                };
+    GeoServerTemplateLoader templateLoader =
+        new GeoServerTemplateLoader(clazz, resoureLoader) {
+          @Override
+          public Object findTemplateSource(String path) throws IOException {
+            Object source = null;
+
+            APIService service = clazz.getAnnotation(APIService.class);
+            if (service != null) {
+              source = super.findTemplateSource(service.landingPage() + "/" + path);
```

Comment:
Ok so it seems this would make it at "ogcapi/features" right?
I don't see a test, given it's a community module, it's not require either, but... did you try it out manually and verified it can be customized? 
Some docs would help user finding this possibilty... however the module has none so far.

Reply:
Ah, found the docs in the other PR.  Assuming you also tried it out, juding from the docs.

Reply:
Yes it is working fine, and you found the docs.

Reply:
Good:

* This allows for `templates/ogc/features/landingPage.ftl` to be customized.

Bad: 

* It does not fix the issue of someone does add `templates/landingPage.ftl` all the services will find it and will break :(

Reply:
Agreed